### PR TITLE
Profile page menu

### PR DESCRIPTION
**Pull request by mjoyce91**
Adds a side-nav/menu to the Profile page based on wireframes. Keeps profile page otherwise untouched for the most part, with the exception of adding a `max-height` and `overflow-y` to work well with the new menu.

### CODE REVIEW

File: src/Components/ProfileMenu/ProfileMenu.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import NavLinksContainer from './NavLinksContainer';
+import NavLink from './NavLink';
+
+const ProfileMenu = ({ currentPath }) => (
+  <div className="usa-grid-full profile-menu">
+    <div className="menu-title">
+      Menu
+    </div>
+    <NavLinksContainer>
+      <NavLink title="Home" iconName="home" link="/profile/" isHighlighted={currentPath === '/profile/'} />
```
Comment (burgwyn) [inline src/Components/ProfileMenu/ProfileMenu.jsx]: `currentPath === '/profile/'` strikes me as something that could be bug-prone

Comment (mjoyce91) [reply src/Components/ProfileMenu/ProfileMenu.jsx]: Yeah, something I just noticed is if you navigate directly to `/profile/searches` (no trailing slash), then it doesn't highlight. `Link` has a built in prop to do this, but I need to perform it to the parent container.

Comment (burgwyn) [reply src/Components/ProfileMenu/ProfileMenu.jsx]: We may want to keep a `Map` in the parent to ensure unique values for the links

Comment (mjoyce91) [reply src/Components/ProfileMenu/ProfileMenu.jsx]: https://github.com/ReactTraining/react-router/blob/master/packages/react-router/modules/matchPath.js

**Pull request merged**